### ISSUE TITLE
Customise effects?

### ISSUE THREAD

**Issue by TimmyTwoHatsTTV** [racerxdl/minecrowdcontrol#12]
Is it possible to change/create our own custom effects for this mod?

**Comment by racerxdl** [comment]
Yes, that's possible. It will not work directly on crowd control (you can use their test SDK to test) but if you get any custom effects, you can make a PR here and I will talk to them to update their client to add the effect ^^

**Comment by TimmyTwoHatsTTV** [comment]
Very cool! 
Are there any tutorials on how to add custom effects or anything like that? I have little to no coding experience and I was checking out some of the files the other day and I honestly would have no idea where to start at this stage. 
Or is it possible to request certain effects here so someone who can actually code can just get in and get it done? haha.

**Comment by racerxdl** [comment]
Well, its somewhat hard to explain how to make effects because the Minecraft API is a reverse engineering that it took me a while to at least do something. But you can ask for effects here and we can implement it if possible ^^
